docker/test/fasttest/run.sh

```diff
@@ -133,6 +133,7 @@ function clone_submodules
             contrib/NuRaft
             contrib/jemalloc
             contrib/replxx
+            contrib/qpl
```

Comment:
The "fast tests" in ClickHouse are smoke tests executing simple SQL statements. To speed these tests up, only a subset of all third-party libraries is checked out as Git submodule (only the ones really needed for simple tests). I would think that DEFLATE/QPI tests can be omitted from this list, 1. because right now, no tests for DEFLATE exist anyways and 2. the DEFLATE codec is rather isolated and unlikely a cause for breakage, respectively unlikely to detect breakage. For testing, it would be sufficient to run tests in the standard "full tests".

Reply:
Done